charmcraft/templates/init-kubernetes/.gitignore.j2

```diff
@@ -0,0 +1 @@
+../init-simple/.gitignore.j2
```

Comment:
so does this syntax mean to use the contents of the other file? i.e., is it a shortcut to Jinja2's `include`?

Reply:
simpler! Jinja2 just sees the file, and for the human checking it it's clearly the same file...